src/Extensions/Mail/IOSendMail.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace IO\Extensions\Mail;
+
+use IO\Services\CustomerPasswordResetService;
+use Plenty\Modules\Plugin\Events\PluginSendMail;
+use Plenty\Modules\Plugin\Services\PluginSendMailService;
+
+
+class IOSendMail
+{
+
+    /**
+     * @param PluginSendMail $pluginSendMail
+     */
+    public function handle(PluginSendMail $pluginSendMail)
+    {
+        $template = $pluginSendMail->getTemplate();
+        $email = $pluginSendMail->getContactEmail();
+
+
+        /** @var PluginSendMailService $pluginSendMailService */
+        $pluginSendMailService = pluginApp(PluginSendMailService::class);
+
+        $response = null;
+        if(strlen($email) > 0)
+        {
+            switch($template)
+            {
+                case 'reset_pw':
```

Comment:
Ist es sinnvoll, die verschiedenen Templates in Konstanten auszulagern? Falls es nicht möglich sein sollte, die Konstanten ins Plugin-Interface zu exportieren, würde ich die in dieser Klasse definieren.

Reply:
jap ist es, baue ich bei den Änderungen mit ein.